test/parallel/test-stream-duplex-from.js

```diff
@@ -144,3 +145,137 @@ const { Duplex, Readable, Writable, pipeline } = require('stream');
     common.mustCall(() => {}),
   );
 }
+
+// Ensure that isDuplexNodeStream was called
+{
+  const duplex = new Duplex();
+  assert.deepStrictEqual(Duplex.from(duplex), duplex);
```

Comment:
It looks like this PR calls `assert. deepStrictEqual()` a number of times. Is it possible to use `assert.strictEqual()` in at least some of these cases?

Reply:
Sure! I've been using the `deepStrictEqual` because I understood that `strictEqual` is for primitive values. So, I think I might misunderstand it. Why use the `assert.strictEqual()` in these cases?

Reply:
@cjihrig just changed the assertions that were primitive values to the strictEqual function. Would you check it?

Reply:
> Why do you use the assert.strictEqual() in these cases?

`strictEqual()` can be used to check if two objects are actually the same object in memory vs. `deepStrictEqual()` comparing the individual properties of the object.

Reply:
Niiiiiice, thank you @Trott and @cjihrig! Just updated the code with it!